app/pages/Auction/BidActionPanel/BidNow.js

```diff
@@ -59,11 +63,19 @@ class BidNow extends Component {
   };
 
   sendBid = async () => {
-    const {sendBid} = this.props;
+    const {sendBid, domain} = this.props;
     const {bidAmount, disguiseAmount} = this.state;
     const lockup = Number(disguiseAmount) + Number(bidAmount);
 
     try {
+      // If the name is not already stored in the wallet, rescan the auction
+      // for past bids before proceeding. The wallet will continue to track
+      // newer bids automatically.
+      if (!domain.walletHasName) {
```

Comment:
Ah, good point. Well, it will only `importName` if this is the first time you bid on an auction. And if you opened the auction it won't happen. I'm imagining that users take advantage of the "rescan auction" button before placing their first bid. This just shims it in anyway if they don't.

I'll look into this approach though, makes sense.

Reply:
Fixed in d2eff90ced3d8e66eba44d0e9f08f7e18112945c